src/FSharpPlus/Operators.fs

```diff
@@ -768,6 +768,12 @@ module Operators =
     /// </summary>
     /// <category index="16">Indexable</category>
     let inline mapi (mapping: 'K->'T->'U) (source: '``FunctorWithIndex<'T>``) : '``FunctorWithIndex<'U>`` = MapIndexed.Invoke mapping source
+   
+    /// <summary>
+    /// Choose with access to the index.
+    /// </summary>
+    /// <category index="16">Indexable</category>
+    let inline choosei (mapping: 'K->'T->'U) (source: '``FunctorWithIndex<'T>``) : '``FunctorWithIndex<'U>`` = ChooseIndexed.Invoke mapping source
```

Comment:
Wrong signature. It should be `U option`

Reply:
Can you add param f and x as you did in the Extensions?
Also, you could possibly change "Choose with access to the index." to "Same as choose but with access ..." or where it corresponds "Same as chooseValues but with ..."

Reply:
Fixed signature and improved all summaries
on operator added params,
on extensions changes to "Same as .."